_data-prepper/pipelines/expression-syntax.md

```diff
@@ -74,58 +45,39 @@ Relational operators are used to test the relationship of two numeric values. Th
 <Number | JSON Pointer> >= <Number | JSON Pointer>
 ```
 
-### Example
+For example, to check if the vlaue of the `status_code` field in an event is within the range of successful HTTTP responses (200 to 299), you can use the following expression:
 
 ```
 /status_code >= 200 and /status_code < 300
 ```
 
-## Equality operators
+### Equality operators
 
-Equality operators are used to test whether two values are equivalent.
+Equality operators are used to test whether two values are equivalent. These operators compare values of any type, including JSON pointers, literals, and expressions. The syntax for using equality operators is as follows: 
 
-### Syntax
 ```
 <Any> == <Any>
 <Any> != <Any>
 ```
 
-### Examples
-```
-/is_cool == true
-3.14 != /status_code
-{1, 2} == /event/set_property
-```
-## Using equality operators to check for a JSON Pointer 
-
-Equality operators can also be used to check whether a JSON Pointer exists by comparing the value with `null`.
+Here are some examples of using equality operators:
 
-### Syntax
-```
-<JSON Pointer> == null
-<JSON Pointer> != null
-null == <JSON Pointer>
-null != <JSON Pointer>
-```
-
-### Example
-```
-/response == null
-null != /response
-```
+- `/is_cool == true`: Checks if the value referenced by the JSON pointer is equal to the Boolean value.
+- `3.14 != /status_code`: Checks if the numeric value is not equal to the value referenced by the JSON pointer.
+- `{1, 2} == /event/set_property`: Checks is the array is equal to the value referenced by the JSON pointer. 
 
-#### Conditional expression
+### Conditional expressions
 
-A conditional expression is used to chain together multiple expressions and/or values.
+Conditional expressions allow you to combine multiple expressions or values using logical operators to create more complex evaluation criteria. The available conditional operators are `and`, `or`, and `not`. The syntax for using these conditional operators is as follows:
 
-#### Syntax
 ```
 <Any> and <Any>
 <Any> or <Any>
 not <Any>
 ```
 
-### Example
+The following are examples of using conditional expressions: 
+
 ```
 /status_code == 200 and /message == "Hello world"
```

Comment:
Should we explain the use like we do for equality operators above?

Reply:
I don't know if this is necessary. It is fairly standard boolean logic.